### ISSUE TITLE
changing PIN freezes after 'Enter passphrase:'

### ISSUE THREAD

**Issue by bereska** [LedgerHQ/app-openpgp#69]
@cslashm merci for the great tool! I have a minor hiccup trying to change the default user PIN. The operation freezes after `Enter passphrase:`
bereska@ubuntuVM:~$ gpg --change-pin
gpg: OpenPGP card no. D2760001240103032C97A0CC32160000 detected

1 - change PIN
2 - unblock PIN
3 - change Admin PIN
4 - set the Reset Code
Q - quit

Your selection? 1
Enter passphrase:

**Comment by cedelavergne-ledger** [comment]
Not reproduced.
Please confirm with latest v2, and open a new issue with more details if you still have the issue
